src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,106 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const { TransactionEvent } = require('forta-agent');
+
+const chainId = 1;
+const votingAddressPromise = getAddress('Voting', chainId);
+
+const { createAlert, handleTransaction } = require('./admin-events');
+
+// TransactionEvent(type, network, transaction, receipt, traces, addresses, block)
+function createTxEvent({ logs, addresses }) {
```

Comment:
forta-agent provides a createTransactionEvent function--it's preferable to use that unless there's a reason not to

Reply:
Done